### ISSUE TITLE
Maybe remove color output from tar_scm?

### ISSUE THREAD

**Issue by jengelh** [openSUSE/obs-service-tar_scm#213]
Color codes from obs-service-tar_scm is copied verbatim into OBS status messages. I am impartial; the message is still readable, which is good enough for me. What do people think about removing (or keeping) the color codes?

# osc r -v
Debian_9.0           x86_64     broken: service error: &#x1b;[31mThe chosen parent_tag is not an ancestor of the checked out commit. Cannot compute a (meaningful) tag offset.&#x1b;[0m

(This error messages comes from PR #211 as I am running my own dogfood.)

**Comment by M0ses** [comment]
IIRC - tar_scm is not adding any color codes and I guess osc is doing neither. I would guess the git output contains the color codes from the beginning. 

Maybe you also could solve your issue with a modification of your git config - color.* seems to be promising to me:

https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration

**Comment by jengelh** [comment]
Nah, I should just make tar_scm do a isatty test, because (as I found out once again) it was myself who added the color codes.

**Comment by aspiers** [comment]
[**@M0ses**](https://github.com/M0ses) commented on [19 Apr 2018, 20:09 BST](https://github.com/openSUSE/obs-service-tar_scm/issues/213#issuecomment-382848392 "2018-04-19T19:09:38Z - Replied by Github Reply Comments"):
> IIRC - tar_scm is not adding any color codes

Actually it is, e.g.

https://github.com/openSUSE/obs-service-tar_scm/blob/c15f106774db64ac32bc404da5d07b8a27f837c2/TarSCM/scm/git.py#L196

[**@jengelh**](https://github.com/jengelh) commented on [19 Jun 2018, 12:30 BST](https://github.com/openSUSE/obs-service-tar_scm/issues/213#issuecomment-398366861 "2018-06-19T11:30:29Z - Replied by Github Reply Comments"):
> Nah, I should just make tar_scm do a isatty test, because (as I found out once again) it was myself who added the color codes.

Yeah, adding an `isatty` test sounds like a good option.

**Comment by jengelh** [comment]
I have an update, though it's based on the PR for #211 , so I'm waiting until that has gotten through.

**Comment by aspiers** [comment]
Thanks for the nudge - I'm not getting notifications from GitHub when new commits are pushed to PRs I'm watching, so I hadn't realised you'd updated it.  Not sure how to fix that.
